go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/redhat-developer/gitops-operator
 go 1.16
```

Comment:
Are we upgrading go version here?

Reply:
There is a different PR for upgrading go version https://github.com/redhat-developer/gitops-operator/pull/307